compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/NodeFinder.java

```diff
@@ -1522,4 +1565,33 @@ private boolean isClassForService(TopLevelNode node) {
         return ((BLangClassDefinition) node).flagSet.contains(Flag.SERVICE) && ((BLangClassDefinition) node).flagSet
                 .contains(Flag.ANONYMOUS);
     }
+
+    private boolean isWithinNodeMetaData(TopLevelNode node) {
+        if (node instanceof AnnotatableNode) {
+
+            List<AnnotationAttachmentNode> nodes =
+                    (List<AnnotationAttachmentNode>) ((AnnotatableNode) node).getAnnotationAttachments();
+
+            for (AnnotationAttachmentNode annotAttachment : nodes) {
+                if (PositionUtil.isRangeWithinNode(this.range, annotAttachment.getPosition())) {
+                    return true;
+                }
+            }
+        }
+
+        if (node instanceof DocumentableNode) {
+            BLangMarkdownDocumentation markdown = ((DocumentableNode) node).getMarkdownDocumentationAttachment();
+            if (markdown != null) {
+                LinkedList<BLangMarkdownParameterDocumentation> parameters = markdown.getParameters();
+                for (BLangMarkdownParameterDocumentation parameter : parameters) {
+                    if (PositionUtil.isRangeWithinNode(this.range, parameter.getPosition())) {
+                        return true;
```

Comment:
Can you please check why this is not watched by Codecov?

Reply:
After an offline discussion, we realized that `NodeFinder` does not have a requirement to determine the type in markdown documentation. Hence, we removed this implementation from the `NodeFinder`.